plugins/cue_points/base/lib/kCuePointManager.php

```diff
@@ -448,10 +452,23 @@ public static function postProcessCuePoints( $liveEntry, $cuePointsIds = null )
 	}
 
 	/**
-	 * @param LiveEntry $liveEntry
+	 * @param mediaInfo $mediaInfo
 	 */
-	public static function copyCuePointsFromLiveToVodEntry( $vodEntry )
+	public static function copyCuePointsFromLiveToVodEntry( $mediaInfo )
 	{
+		$flavorAssetId = $mediaInfo->getFlavorAssetId();
+		$flavorAsset = assetPeer::retrieveById( $flavorAssetId );
+		if ( ! $flavorAsset || ! $flavorAsset->hasTag(assetParams::TAG_RECORDING_ANCHOR) )
+		{
+			return;
+		}
+
+		$vodEntry = $flavorAsset->getentry();
+		if ( ! $vodEntry || $vodEntry->getSourceType() != EntrySourceType::RECORDED_LIVE )
```

Comment:
@AviBueno, no need to check the source type if you already checked the recording anchor tag.